package.json

```diff
@@ -1,48 +1,39 @@
 {
   "name": "purescript-web3-example",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "directories": {
-    "test": "test"
-  },
+  "private": true,
   "scripts": {
-    "psc-package": "psc-package update",
-    "postinstall": "npm run psc-package && npm run generator",
-    "generator": "./node_modules/.bin/pulp build --src-path generator -m Generator --to generator.js && node generator.js --abis contracts/abis --dest src; rm generator.js",
-    "build": "./node_modules/.bin/pulp browserify --to dist/app.js",
-    "build-css": "node-sass styles/ -o dist/",
-    "watch-css": "npm run build-css && node-sass styles/ -o dist/ --watch --recursive",
-    "webpack": "./node_modules/.bin/webpack --config ./webpack.config.js --progress --profile --colors",
-    "webpack-dev-server": "webpack-dev-server --content-base dist --progress"
+    "generator": "pulp build --src-path generator -m Generator --to generator.js && node generator.js --abis contracts/abis --dest src; rm generator.js",
+    "webpack-dev-server": "webpack-dev-server --content-base dist --progress",
+    "build": "pulp browserify --to dist/app.js",
+    "webpack": "webpack --config ./webpack.config.js --progress --profile --colors"
   },
   "dependencies": {
     "bignumber.js": "git+https://github.com/frozeman/bignumber.js-nolookahead.git",
     "create-react-class": "^15.6.0",
     "crypto-js": "^3.1.8",
+    "ethjs-provider-http": "^0.1.6",
     "http-serve": "^1.0.1",
-    "material-ui": "^0.18.7",
+    "material-ui": "^0.19.0",
```

Comment:
This way we can revert this version update 
https://github.com/mui-org/material-ui/issues/10516#issuecomment-370440240